### DISCUSSION TITLE
Job runner did'nt run for 1st Feb

### DISCUSSION THREAD

**Discussion by slice-akhil** [jobrunr/jobrunr#927] [General]
Hey Guyz need some help. Actually we use job runner for executing job but surprisingly it did'nt ran for complete 1st feb and automatically started running on 2nd feb. There are no logs and error in the system not sure how to debug this further ?
![Uploading Screenshot 2024-02-03 at 12.10.38 AM.png…]()

**Comment by auloin** [comment]
HI @slice-akhil, we cannot help with the amount of information you're giving us. Please add more details (e.g., are the jobs scheduled, recurring, what are the schedule expressions, ...).

**Reply by rdehuyss** [reply]
Please also update your Github profile (e.g. name, company you are working for, ...)!
